content/technology/solutions/altinn-platform/receipt/_index.nb.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Kvittering
+description: Kvitteringskomponenten i Altinn Platform viser kvittering for apper.
+tags: [platform, register]
+---
+
+Kvitteringskomponenten er laget for å vise en kvittering for alle apper. Noen apper vil fjernes fra Altinn Apps med tiden, men innsendte data vil likevel være tilgjengelig, og Kvitteringskomponenten er ansvarlig for å vise en generisk kvittering for innsendte data uavhengig av applikasjonen.
+
+Tekster i Kvitteringskomponenten kan overstyres av applikasjonen ved å spesifisere tekster i applikasjonens `config/texts/resource.xx.json` fil.
+
+
+{{%notice info%}}
+Overstyring av tekster i kvitteringen vil ha påvirkning for alle kvitteringer for den gitte applikasjonen. Dette betyr at alle skjemaer som allerede er insendt vil også få det oppdaterte tekstene. PDF filen som er generert vil IKKE påvirkes av dette, men denne vil inneholde tekstene som var tilgjengelig når PDFen ble generert.
+{{% /notice%}}
+
+
+Dette er tekstnøklene som kan brukes for å overstyre standardtekstene:
+
+```
+receipt_platform.attachments
```

Comment:
Just a "feature request" from me, could we add a screenshot indicating which key is placed where? Or is this intuitive?

Reply:
Good idea, Ill add it